manager/director/apps/docs/utils.py

```diff
@@ -63,7 +63,6 @@
 cleaner = bleach.sanitizer.Cleaner(
     tags=ALLOWED_TAGS,
     attributes=ALLOWED_ATTRS,
-    styles=ALLOWED_STYLES,
```

Comment:
When I Ctrl + F for 'ALLOWED_STYLES' I don't see any other code adding to the list or declaring it other than [line 61 of utils.py](https://github.com/tjcsl/director4/blob/master/manager/director/apps/docs/utils.py#L61), which just declares it as an empty list. But, just in case I've replaced it with `css_sanitizer` in the latest commit

Reply:
note: i bumped bleach to latest